Backend/app/provider-platform/dynamic-offer-driver-drainer/src/DBSync/Create.hs

```diff
@@ -176,16 +189,55 @@ runCreateCommands cmds streamKey = do
           EL.logError ("Create failed: " :: Text) (show cmdsToErrorQueue <> "\n Error: " <> show x :: Text)
           pure [Left entryIds]
 
-streamDriverDrainerCreates :: ToJSON a => Producer.KafkaProducer -> [a] -> Text -> IO (Either Text ())
-streamDriverDrainerCreates producer dbObject streamKey = do
-  let topicName = "driver-drainer"
-  result' <- mapM (KafkaProd.produceMessage producer . message topicName) dbObject
-  if any isJust result' then pure $ Left ("Kafka Error: " <> show result') else pure $ Right ()
+streamDriverDrainerCreates ::
+  ( ToJSON (table Identity)
+  ) =>
+  Producer.KafkaProducer ->
+  Text ->
+  Bool ->
+  [(table Identity, b, EL.KVDBStreamEntryID, DBCreateObject)] ->
+  Text ->
+  IO (Either Text ())
+streamDriverDrainerCreates producer tableName pushToS3 dbObject streamKey = do
+  result' <- forM dbObject $ \obj -> do
+    result'' <- KafkaProd.produceMessage producer . message $ obj
+    flushResult <- timeout (5 * 60 * 1000000) $ KafkaProd.flushProducer producer
```

Comment:
this `flushProducer` function was removed earlier, but I faced error while testing`KafkaResponseError RdKafkaRespErrQueueFull`, then bring it back and all worked fine